andela/gimli-rc
# #147425815 Make the api endpoints restful

#### What does this PR do?
This PR adds the feature to enable API Endpoints to be RESTFUL

#### Description of Task to be completed?
N/A

#### How should this be manually tested?
This can be tested using any RESTFUL API app that allows you to access api routes, we will use POSTMAN in this test description,
We will make a GET request to fetch Products , type in the following: 
`GET  http://localhost:3000/publications/Products`

and the 

> RESPONSE
can be see in the screenshot below

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#147425815

#### Screenshots (if appropriate)
<img width="1177" alt="screen shot 2017-07-01 at 2 45 10 pm" src="https://user-images.githubusercontent.com/28437833/27763010-c42904c6-5e73-11e7-9492-c21fc6c7e02f.png">

#### Questions:
